components/admin/permissions/permission-checkbox.tsx

```
@@ -0,0 +1,103 @@
+          >
+            {permissionDescription}
+          </div>
+        )}
+
+        {/* Accessibility: Show permission key on hover for developers */}
+        {isHovered && process.env.NODE_ENV === 'development' && (
+          <div className="text-xs text-blue-500 dark:text-blue-400 mt-1 font-mono">
+            {permission}
+          </div>
+        )}
+      </label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)